api/src/exceptions/range-not-satisfiable.ts

```diff
@@ -1,9 +1,12 @@
 import { BaseException } from '@directus/shared/exceptions';
+import { Range } from '@directus/drive';
 
 export class RangeNotSatisfiableException extends BaseException {
-	constructor(rangeValue: string) {
+	constructor(range?: Range) {
+		const rangeString = range && (range?.start || range?.end) ? `"${range.start ?? ''}-${range.end ?? ''}" ` : '';
```

Comment:
`range.start` can be `0` so you should compare with `== null` to check if is `null` or `undefined`